src/components/icons/icons.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { SCVWrapperVaraints } from "./SVGWrapper/SVGWrapper.css";
+import * as icons from "./index";
+
+const macawIcons = Object.values(icons);
+
+export default {
+  title: "components/Icons",
+};
+
+export const Default = () =>
+  ["Small", "Medium", "Large"].map((size) => (
+    <>
+      <h3>{size}</h3>
+      <div
```

Comment:
```suggestion
      <Box
```

What about using `<Box/>` here as well?